cmd/clef/main.go

```diff
@@ -519,23 +546,30 @@ func readMasterKey(ctx *cli.Context) ([]byte, error) {
 	if err := checkFile(file); err != nil {
 		return nil, err
 	}
-	masterKey, err := ioutil.ReadFile(file)
+	cipherKey, err := ioutil.ReadFile(file)
 	if err != nil {
 		return nil, err
 	}
-	if len(masterKey) < 256 {
-		return nil, fmt.Errorf("master key of insufficient length, expected >255 bytes, got %d", len(masterKey))
+
+	password := getPassPhrase("Unlocking master seed of clef", false)
```

Comment:
Make an ui-method is cumbersome, maybe we could put `readMasterKey` before ui setup, get the password from stdin first to avoid the problem, WDYT?

Reply:
The problem with that is that it's only in some scenarions that you will need a password -- in the 'naive' usecase where no rules are used, there's no need for the user to input a password. So it is a bit of a 'hack' to basically say, to UI implementors: " If you start clef with `--rules=...`, then you need to pass the password over `stdin`, but not as part of a rpc message, just raw data" -- and additionally, it will mess with the RPC server which is listening on `stdin`.